scopegen/scopegen.go

```diff
@@ -51,6 +53,11 @@ func (s *scopePlugin) Generate(req *plugin_go.CodeGeneratorRequest) (*plugin_go.
 				}
 			case "out":
 				baseDir = v
+			case "output_version":
+				var err error
+				if outputVersion, err = strconv.Atoi(v); err != nil {
+					return nil, fmt.Errorf("unknown output version: err=%s", err.Error())
```

Comment:
nit: not `:%w`?

Reply:
`%w` is too new, we are still support go 1.11.